services/one-stream/package/buildAnyPackage.js

```diff
@@ -140,7 +142,17 @@ export const buildAnyPackage = async (packageId, config) => {
           anEvent.STATE_PLAN.SPW_STATUS_ID,
           seaToolStatus
         );
-        putParams.Item.currentStatus = SEATOOL_TO_ONEMAC_STATUS[seaToolStatus];
+        seaToolStatus &&
+          (putParams.Item.currentStatus =
+            SEATOOL_TO_ONEMAC_STATUS[seaToolStatus]);
+        if (
+          anEvent.STATE_PLAN.PROPOSED_DATE &&
+          typeof anEvent.STATE_PLAN.PROPOSED_DATE === "number"
+        )
+          putParams.Item.proposedEffectiveDate = DateTime.fromMillis(
+            anEvent.STATE_PLAN.PROPOSED_DATE
+          ).toFormat("yyyy-LL-dd");
+        else putParams.Item.proposedEffectiveDate = "none";
```

Comment:
I know this was talked about but I wasnt clear on all the details. So if a record comes in with an empty, nonnumeric, or nonexistent PROPOSED_DATE field then we overwrite existing value and update to "none"?
I didnt catch the business reason but I guess there is a need to allow them to intentionally clear out the field?

Reply:
so, I guess it does give them the ability to clear out the field... but the reason is that we can't be in the business of correcting SEA Tool - if they lose it or change it or clear it... that's on them :)